SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/OntologyMapper.Mapped.csproj

```diff
@@ -29,6 +29,6 @@
 
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.2" />
-		<PackageReference Include="Microsoft.SmartPlaces.Facilities.OntologyMapper" Version="0.6.5-preview" />
+		<PackageReference Include="Microsoft.SmartPlaces.Facilities.OntologyMapper" Version="0.6.6-preview" />
```

Comment:
@jbkoh Actually the change that needs to be made here is in the following two fields:

		<AssemblyVersion>0.4.10</AssemblyVersion>
		<PackageVersion>0.4.10-preview</PackageVersion>

Reply:
Thanks for the pointer. Could you check this again? https://github.com/Azure/opendigitaltwins-tools/pull/98/commits/a611d3bc860fe8777933cc51675cfe01e0915d6c